apps/services/auth/admin-api/src/environments/environment.ts

```diff
@@ -20,7 +20,8 @@ const prodConfig = {
   },
   auth: {
     audience,
-    issuer: process.env.IDENTITY_SERVER_ISSUER_URL,
+    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+    issuer: process.env.IDENTITY_SERVER_ISSUER_URL!,
```

Comment:
out of scope for this PR perhaps, but we could probably typeguard this with a config/env loader

Reply:
Yeah, there's still plenty of legacy configuration in these environment files which I'd love to get rid off. 😅

But out of scope here.